docs/translations.rst

```diff
@@ -0,0 +1,244 @@
+    - `How setup this file for babel configurations <http://babel.pocoo.org/en/latest/setup.html>`_.
+
+Extraction Configurations
+-------------------------
+
+First of all you have to get into the folder where you have your Sphinx theme and create a mapping file 
+called :file:`babel.cfg` that contains the **extraction from Jinja2 HTML templates** configurations. 
+For typical Sphinx extensions, this is what you want in there:
+
+.. code:: cfg
+
+    # Extraction from Jinja2 HTML templates
+    [jinja2: **/**.html]
+    encoding = utf-8
+    ignore_tags = script,style
+    include_attrs = alt title summary placeholder
+
+
+.. seealso::
+
+    More details check out the following links:
+
+    - `How setup this file <http://babel.pocoo.org/en/latest/setup.html>`_
+    - `A previous file example description <http://babel.pocoo.org/en/latest/messages.html#extraction-method-mapping-and-configuration>`_
+
+Administrative Tasks
+====================
+
+The ``babel`` package have a *Distutils/Setuptools Integration* which supports the options 
+defined in the :file:`setup.cfg` file that can be executed via command line.
+
+These options are the commonly using as **"Translations Administrative Tasks"** and the most 
+used tasks are described below:
+
+Extract messages
+----------------
+
+It can extract localizable messages from a variety of difference source files, 
+and generate a PO (portable object) template file from the collected messages.
+
+Running the following command will produce a PO template file:
+
+.. code:: bash
+
+    $ python ./setup.py extract_messages -o ./sphinx_rtd_theme/locale/sphinx.pot
+
+.. tip::
+
+    More options please, check out http://babel.pocoo.org/en/latest/setup.html#extract-messages
+
+Init catalog
+------------
+
+It creates a new translation catalog based on a PO template file (POT). Running the following 
+command will produce a PO file:
+
+.. code:: bash
+
+    $ python ./setup.py init_catalog -l es -i ./sphinx_rtd_theme/locale/sphinx.pot \
+                                     -o ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po
+
+.. tip::
+
+    More options please, check out http://babel.pocoo.org/en/latest/setup.html#init-catalog
+
+Update catalog
+--------------
+
+It updates an existing translations catalog based on a PO template file (POT). Running the following 
+command will update a PO file:
+
+.. code:: bash
+
+    $ python ./setup.py update_catalog -l es -i ./sphinx_rtd_theme/locale/sphinx.pot \
+                                             -o ./sphinx_rtd_theme/locale/es/LC_MESSAGES/sphinx.po
+
+.. tip::
+
+    More options please, check out http://babel.pocoo.org/en/latest/setup.html#update-catalog
+
+Compile catalog
+---------------
+
+It compile catalog an existing translations based on PO files into MO files. Running the following 
+command will compile catalog of PO files:
+
+.. code:: bash
+
+    $ python ./setup.py compile_catalog -d ./sphinx_rtd_theme/locale/
```

Comment:
@agjohnson I used Babel because "Sphinx uses Babel to extract messages and maintain the catalog files." check out the following link http://www.sphinx-doc.org/en/master/devguide.html#locale-updates

I trying to improvement the i18n support to the "sphinx_rst_theme" package because I am Plone Developer and Trainer, and time ago the official Plone training iniciative usa the "sphinx_rst_theme" package as theme into the training HTML recourses. Then I am working to translate all this training to Spanish but the "sphinx_rst_theme" package have some HTML template that don't have support for i18n.

This is the main reason for my improvements of "sphinx_rst_theme" package, my logic said me that Sphinx using babel for extract and manage the string to translate, thene I need to adopts the same domain "sphinx" and the same library for extract and manage the new translation inside this theme package.

Now yo can undestand me why don't use any integration with Django, because I need use the "sphinx_rst_theme" package has a Sphinx theme for generate HTML documentations initialy.